genesyscloud/process_automation_trigger/resource_genesyscloud_processautomation_trigger.go

```diff
@@ -329,10 +349,22 @@ func buildTarget(d *schema.ResourceData) *Target {
 			targetType := targetMap["type"].(string)
 			id := targetMap["id"].(string)
 
-			return &Target{
+			target := &Target{
 				Type: &targetType,
 				Id:   &id,
 			}
+
+			workflowTargetSettingsInput := targetMap["workflow_target_settings"].(*schema.Set).List()
+
+			if len(workflowTargetSettingsInput) > 0 {
+				workflowTargetSettingsInputMap := workflowTargetSettingsInput[0].(map[string]interface{})
```

Comment:
Sorry, one more thing. Because `workflow_target_settings` doesn't have any required fields, the end user could define the map like this: `workflow_target_settings {}` in their terraform config. In this case, the slice  `workflowTargetSettingsInput` will contain one nil item, and we'll get a panic here. 

To prevent this you just need to confirm the type casting was successful, like this:

```golang
workflowTargetSettingsInputMap, ok := workflowTargetSettingsInput[0].(map[string]interface{})
if !ok {
  return target 
}
```

Reply:
I tried out what you suggested but tha didnt quite seem to do it. I added the following and it seems to be working.
`if dataFormat == "" {
					return target
				}`